assets/src/domain/shared/entities/tickets/predicates/isExpired/index.test.ts

```diff
@@ -0,0 +1,46 @@
+import { formatISO } from 'date-fns';
+import { add, sub } from '../../../../../../application/services/utilities/date';
+
+import isExpired from './index';
+import { nodes as tickets } from '../../../../../eventEditor/services/apollo/queries/tickets/test/data';
+
+describe('isExpired', () => {
+	it('should return false if ticket is trashed', () => {
+		tickets.forEach((ticket) => {
+			const newTicket = { ...ticket, isTrashed: true };
+			expect(isExpired({ ticket: newTicket })).toEqual(false);
```

Comment:
`.toBe()` is better when not intending to check deep equality, e.g. for objects. So for non-object types, `.toBe()` is faster.

Reply:
Done: https://github.com/eventespresso/event-espresso-core/pull/2380/commits/fc5500bd30dc86f19ca0d92b99792e842b720c28